docs/content/preview/releases/ybdb-releases/v2.21.md

```diff
@@ -365,9 +365,9 @@ For more information, refer to the [YugabyteDB Kubernetes Operator](https://gith
 
 * [New Kubernetes Operator](https://github.com/yugabyte/yugabyte-k8s-operator). Automated deployment and management of clusters via the Kubernetes operator pattern. Includes support for YugabyteDB universes as a Kubernetes custom resource. Backup, upgrade, scale-out, scale-in, and more are possible on this Kubernetes custom resource. {{<badge/tp>}}
 
-* [YSQL: DDL concurrency](../../../reference/configuration/yb-tserver/#ysql-enable-db-catalog-version-mode). Support for isolating DDLs per database. Specifically, a DDL in one database does not cause catalog cache refreshes or aborts transactions due to breaking change in another database. {{<badge/tp>}}
+* [YSQL: DDL concurrency](../../../reference/configuration/yb-tserver/#ddl-concurrency-flags). Support for isolating DDLs per database. Specifically, a DDL in one database does not cause catalog cache refreshes or aborts transactions due to breaking change in another database. {{<badge/tp>}}
 
-* YSQL: DDL atomicity. Ensures that YSQL DDLs are fully atomic between YSQL and DocDB layers, that is in case of any errors, they are fully rolled back, and in case of success they are applied fully. Currently, such inconsistencies are rare but can happen. {{<badge/tp>}}
+* [YSQL: DDL atomicity](../../../reference/configuration/yb-tserver//#ddl-atomicity-flags). Ensures that YSQL DDLs are fully atomic between YSQL and DocDB layers, that is in case of any errors, they are fully rolled back, and in case of success they are applied fully. Currently, such inconsistencies are rare but can happen. {{<badge/tp>}}
```

Comment:
```suggestion
* [YSQL: DDL atomicity](../../../reference/configuration/yb-tserver/#ddl-atomicity-flags). Ensures that YSQL DDLs are fully atomic between YSQL and DocDB layers, that is in case of any errors, they are fully rolled back, and in case of success they are applied fully. Currently, such inconsistencies are rare but can happen. {{<badge/tp>}}
```